libs/block-template-html/block-metadata.json

```diff
@@ -11,7 +11,7 @@
   },
   "protocol": "0.2",
   "displayName": "Template",
-  "examples": [{ "name": "World" }],
+  "examples": [{ "http://example.com/types/property-type/name/": "World" }],
```

Comment:
Could use the same as in our other templates, which is a real URL

Reply:
Happy to do that now

Reply:
[Remove redundant files, use real types](https://github.com/blockprotocol/blockprotocol/pull/1021/commits/20ab0d7034706eb82f4e916d10e57f3563a5387e)